src/utils/isTag.js

```diff
@@ -2,5 +2,5 @@
 import type { Target } from '../types';
 
 export default function isTag(target: Target) /* : %checks */ {
-  return typeof target === 'string';
+  return typeof target === 'string' && target.charAt(0) === target.charAt(0).toLowerCase();
```

Comment:
let's wrap this in a `NODE_ENV !== 'production'` check

Reply:
I think it's not necessary. Built-in tag components are always lower-cased by [react spec](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)

Reply:
Yes, I just don't want to include the code in production since it will never happen :)